src/components/TopBar.js

```diff
@@ -7,28 +7,29 @@ import Text from './Text';
 import Colors from '../styles/Colors';
 
 class TopBar extends Component {
-
-    constructor(props) {
-        super();
-    }
-
+    
+    onClick = () => {
+        const {loggedUser, login, logout} = this.props;
+        const onPressAction = loggedUser ? logout : login;
+        onPressAction();
+    };
+    
     render() {
-        const { loggedUser, login, logout } = this.props;
-        const buttonAction = loggedUser ? logout : login;
+        const {loggedUser} = this.props;
         const buttonText = loggedUser ? 'Wyloguj' : 'Zaloguj';
         return (
             <TopBarWrapper>
                 <Text type="basic" color={ Colors.white } display={ 'inline' }>
                     Zaloguj się przez GitHub
                 </Text>
-                <Button marginLeft='10px' onClick={ buttonAction }
+                <Button marginLeft='10px' onClick={this.onClick}
```

Comment:
This line is sad without spaces :(

Reply:
why @kgajowy why 😢

Reply:
We should force this style to let developers work like they want ;) I will make a prettier+husky integration in the next PR!

Reply:
maybe try do it in that branch, hmm?

Reply:
no, rather not - feature branches <3